openmdao/code_review/test_lint_docstrings.py

```diff
@@ -495,95 +504,200 @@ def check_function(self, dir_name, file_name, func_name, func, failures):
                 failures[key] = new_failures
 
     def test_docstrings(self):
-        print_info = False
-
         failures = {}
 
         # Loop over directories
-        for dir_name in directories:
-            dirpath = dir_name
-            if print_info:
-                print('-'*len(dir_name))
-                print(dir_name)
-                print('-'*len(dir_name))
+        for dirpath in sorted(directories):
 
             # Loop over files
-            for file_name in os.listdir(dirpath):
-                if not file_name.startswith("_") and file_name[-3:] == '.py' and not os.path.isdir(file_name):
-                    if print_info:
-                        print(file_name)
+            for file_name in sorted(os.listdir(dirpath)):
+                if not file_name.startswith("_") and file_name[-3:] == '.py' \
+                   and not os.path.isdir(file_name):
 
-                    # to construct module name, remove part of abs path that
-                    # precedes 'openmdao', and then replace '/' with '.' in the remainder.
-                    mod1 = re.sub(r'.*openmdao', 'openmdao', dir_name).replace('/', '.')
+                    # To construct module name, remove part of abs path that
+                    # precedes 'dymos', and then replace '/' with '.' in the remainder.
```

Comment:
You can change this from dymos to openmdao.

Reply:
Fixed